spec/project/changelog_spec.rb

```diff
@@ -31,16 +31,16 @@
   end
 
   describe 'entry' do
-    subject(:entries) { lines.grep(/^\*/).map(&:chomp) }
+    subject(:entries) { lines.grep(/^-/).map(&:chomp) }
 
     let(:lines) { changelog.each_line }
 
     it 'has a whitespace between the * and the body' do
-      expect(entries).to all(match(/^\* \S/))
+      expect(entries).to all(match(/^- \S/))
     end
 
     it 'has a link to the contributors at the end' do
-      expect(entries).to all(match(/\(\[@\S+\]\[\](?:, \[@\S+\]\[\])*\)$/))
+      expect(entries).to all(match(/\(\[@\S+\](?:, \[@\S+\])*\)$/))
     end
 
     describe 'link to related issue on github' do
```

Comment:
We have exactly _one_ such issue in the Changelog. Shouldn’t we just delete this example group?

Reply:
You're right, we don't need that example group. I deleted it.